cliconfig/config.go

```diff
@@ -32,7 +32,7 @@ var (
 func getDefaultConfigDir(confFile string) string {
 	confDir := filepath.Join(homedir.Get(), confFile)
 	// if the directory doesn't exist, maybe we called docker with sudo
-	if _, err := os.Stat(configDir); err != nil {
+	if _, err := os.Stat(confDir); err != nil {
```

Comment:
this is probably the right change, but having the word "File" in the "confFile" parameter is really confusing because sometime this func is called with a dir name not a file name.  Not your fault, just noticed it while trying to understand the flow.